adapters/build.gradle

```diff
@@ -23,7 +23,7 @@ configurations {
 
 dependencies {
     compile 'com.android.support:appcompat-v7:23.3.0'
-
+    compile 'com.android.support:recyclerview-v7:23.3.0'
```

Comment:
`require` instead of `compile`? I guess we don't have strong dependency on recyclerview version?